spec/jobs/rapid_ready_for_decision/form526_base_job_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'sidekiq/testing'
+
+RSpec.describe RapidReadyForDecision::Form526BaseJob, type: :worker do
+  before { Sidekiq::Worker.clear_all }
+
+  let!(:user) { FactoryBot.create(:disabilities_compensation_user, icn: '2000163') }
+  let(:submission) { create(:form526_submission, :with_uploads, user: user, submitted_claim_id: '600130094') }
+
+  let(:mocked_observation_data) do
+    [{ effectiveDateTime: "#{Time.zone.today.year}-06-21T02:42:52Z",
+       practitioner: 'DR. THOMAS359 REYNOLDS206 PHD',
+       organization: 'LYONS VA MEDICAL CENTER',
+       systolic: { 'code' => '8480-6', 'display' => 'Systolic BP', 'value' => 115.0, 'unit' => 'mm[Hg]' },
+       diastolic: { 'code' => '8462-4', 'display' => 'Diastolic BP', 'value' => 87.0, 'unit' => 'mm[Hg]' } }]
+  end
```

Comment:
Hmm I'm not seeing this used in this particular test. Is it for more tests in downstream PRs?

FWIW: other tests use the Lighthouse class + VCR to make this data available.

Reply:
Yes, the tests that use this are restored/added in Part 3 PR #9597: https://github.com/department-of-veterans-affairs/vets-api/pull/9597/files#diff-08179f4e3d33c9ef93c77850c81934f28fc825ed339ea260804a04b10ca31287R48.

This code was copied from `disability_compensation_job_spec.rb:29` -- trying to minimize changes. We can make it more consistent in another PR.